src/dao.rs

```diff
@@ -131,6 +135,23 @@ impl StatusDao {
         .await?;
         Ok(())
     }
+
+    pub async fn standard_score(
+        &self,
+        role_id: &str,
+        node_id: &str,
+    ) -> sqlx::Result<StandardScore> {
+        let standard_score = sqlx::query_as!(
+            StandardScore,
+            r#"SELECT CALC.STANDARD_SCORE($1, $2)"#,
```

Comment:
function expects role_id as database format: `R`, `P`. we have to remap from url to databse ormat.